app/src/main/java/com/github/scribeWizTeam/scribewiz/NotesDisplayedActivity.kt

```diff
@@ -0,0 +1,51 @@
+package com.github.scribeWizTeam.scribewiz
+
+import android.net.Uri
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import alphaTab.AlphaTabView
+import alphaTab.Settings
+import alphaTab.core.ecmaScript.Uint8Array
+import alphaTab.importer.ScoreLoader
+import alphaTab.model.Score
+import alphaTab.synth.PlayerState
+import android.util.Log
+import android.widget.Toast
+import androidx.activity.result.contract.ActivityResultContracts
+import java.io.ByteArrayOutputStream
+import kotlin.contracts.ExperimentalContracts
+
+
```

Comment:
There could be a few more comments added in between the lines for code readibility.